sources/HUBContentOperationWrapper.m

```diff
@@ -56,29 +58,30 @@ - (void)performOperationForViewURI:(NSURL *)viewURI
                      previousError:(nullable NSError *)previousError
 {
     self.isExecuting = YES;
-    
+
+    id<HUBContentOperationContext> const context = ({
```

Comment:
Aha, cool :smile:

![the more you know](https://media.giphy.com/media/tANpI4H9zlv1u/giphy.gif)